packages/react-native-editor/__device-tests__/pages/editor-page.js

```diff
@@ -823,7 +823,15 @@ class EditorPage {
 				'//android.view.ViewGroup[2]/android.view.ViewGroup/android.view.ViewGroup';
 			await waitForVisible( this.driver, locator );
 
-			await swipeDown( this.driver );
```

Comment:
It is not clear to me why, but this utility failed to close the bottom sheet on Android. Replacing it with a simple tap on the middle-top of the display suffices.